src/__tests__/databaseTests.js

```diff
@@ -1,858 +1,877 @@
+// @flow
```

Comment:
Enable "hide whitespace" to view this PR's diff:

<img width="239" alt="image" src="https://user-images.githubusercontent.com/183747/119332180-88b3f000-bc88-11eb-98b3-bb62f6bc0f20.png">